test/game_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require "./lib/contestant"
+require "./lib/game"
+
+class GameTest < Minitest::Test
+
+  def test_it_exists
+    pick_4 = Game.new('Pick 4', 2)
+
+    assert_instance_of Game, pick_4
+  end
+
+  def test_it_has_readable_attributes
```

Comment:
Technically, `national_drawing?` is its own method and should have its own test.